src/tasks/kube.ts

```diff
@@ -37,17 +37,19 @@ export class KubeTasks {
           for (let i = 1; i <= iterations; i++) {
             // check cheCluster status
             const cheClusterFailState = await this.getCheClusterFailState(namespace)
-            if (cheClusterFailState) {
-              task.title = `${task.title}...failed`
-              throw new Error(`Eclipse Che operator failed, reason: ${cheClusterFailState.reason}, message: ${cheClusterFailState.message}. Consider increasing error recheck timeout with --k8spoderrorrechecktimeout flag.`)
-            }
-
             // check 'PodScheduled' condition
             const podFailState = await this.getPodFailState(namespace, selector, 'PodScheduled')
-            if (podFailState) {
-              // for instance we need some time for pvc provisioning...
+
+            if (cheClusterFailState || podFailState) {
+              // wait some time to recheck
```

Comment:
I think we should do something like:
```ts
await cli.wait(10 * 1000)
i += 10
continue
```
Otherwise, if we face a problem that is resolved in a dozen of seconds, current code will wait whole minute.